app/src/main/kotlin/com/wire/android/ui/calling/controlButtons/CallOptionsControls.kt

```diff
@@ -13,11 +13,15 @@ import androidx.compose.ui.Modifier
 import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.tooling.preview.Preview
 import com.wire.android.R
+import com.wire.android.ui.calling.CallState
 import com.wire.android.ui.theme.wireDimensions
 import com.wire.android.ui.theme.wireTypography
 
 @Composable
-fun CallOptionsControls() {
+fun CallOptionsControls(
+    callState: CallState,
+    toggleMute: () -> Unit
+) {
```

Comment:
how about passing here just a primitive instead of making it depend on CallState ?

Reply:
Good point